tests/assets/scripts/dummy_pmids.txt

```diff
@@ -0,0 +1,2 @@
+PMID_8755636
```

Comment:
When you use `scripts.utils.find_pmids` you can remove these files and instead mock find_pmids